pkg/cmd/server/api/validation/master.go

```diff
@@ -519,9 +519,9 @@ func ValidateRoutingConfig(config api.RoutingConfig) fielderrors.ValidationError
 }
 
 func ValidateAPIServerExtendedArguments(config api.ExtendedArguments) fielderrors.ValidationErrorList {
-	return ValidateExtendedArguments(config, kapp.NewAPIServer().AddFlags)
+	return nil //ValidateExtendedArguments(config, kapp.NewAPIServer().AddFlags)
```

Comment:
Yes it won't be in 1.1.1 because of the rebase.

On Tue, Jan 12, 2016 at 10:44 AM, David Eads <notifications@github.com>
wrote:

> In pkg/cmd/server/api/validation/master.go
> <https://github.com/openshift/origin/pull/6496#discussion_r49470240>:
>
> > @@ -519,9 +519,9 @@ func ValidateRoutingConfig(config api.RoutingConfig) fielderrors.ValidationError
> >  }
> >
> >  func ValidateAPIServerExtendedArguments(config api.ExtendedArguments) fielderrors.ValidationErrorList {
> > -	return ValidateExtendedArguments(config, kapp.NewAPIServer().AddFlags)
> > +	return nil //ValidateExtendedArguments(config, kapp.NewAPIServer().AddFlags)
>
> This is just temporary (why this is WIP) until we can split out the
> upstream code so that depending on the validation here doesn't import
> kubelet/* or kube-codebase/*. Those changes landed and will either be in
> the next rebase or the one after that.
>
> Ok. Edited name to indicate [WIP]. I'd rather hold this until after 1.1.1,
> so we don't lose validation for master-config.yaml files.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/origin/pull/6496/files#r49470240>.
>